apps/storybook/.storybook/preview.ts

```diff
@@ -1,5 +1,4 @@
 import type { Preview } from '@storybook/react';
-import './globals.css';
```

Comment:
hmmm... was this a rebase error? i think this is what i had to add to get tailwind working in storybook... 🤔